homeassistant/components/fritzbox/sensor.py

```diff
@@ -87,6 +82,26 @@ class FritzSensorEntityDescription(
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
         native_value=lambda device: device.power / 1000 if device.power else 0.0,
     ),
+    FritzSensorEntityDescription(
+        key="electric_potential",
+        name="Electric Potential",
+        native_unit_of_measurement=ELECTRIC_POTENTIAL_VOLT,
+        device_class=SensorDeviceClass.VOLTAGE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
+        native_value=lambda device: device.voltage if device.voltage else 0.0,
```

Comment:
based on [API documention (_page 9_)](https://avm.de/fileadmin/user_upload/Global/Service/Schnittstellen/AHA-HTTP-Interface.pdf) `voltage` is returned as 0.001 volts, so needs to be divided by 1000 (_same like `power` property_)